### PR TITLE
Sc 6758

### PR DESCRIPTION
**Pull request by nwhite222**
### Scope of changes

Briefly add notes if applicable and include the story ID

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/locales/en/messages.po [hunk 1]
```diff
@@ -13,6 +13,25 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: src/components/ReviewSubmit/index.tsx:192
+#~ msgid ""
+#~ " for MainNet registration. Your MainNet certificate will be issued upon review by\n"
+#~ "                  the validation team"
+#~ msgstr ""
+#~ " for MainNet registration. Your MainNet certificate will be issued upon review by\n"
+#~ "                  the validation team"
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/en/messages.po:22]: @nwhite222 would you mind removing all the lines that start with `#~`? These have been removed from the web content and should no longer be in the .po files

File: web/gds-user-ui/src/locales/en/messages.po [hunk 2]
```diff
@@ -95,7 +114,7 @@ msgstr "Action"
 msgid "Add Address"
 msgstr "Add Address"
 
-#: src/components/OtherJuridictions/index.tsx:61
+#: src/components/OtherJurisdictions/index.tsx:61
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/en/messages.po:117]: Unfortunately this is a typo in a filepath name that we will have to live with for now - changing it here would not change the filepath and would make it impossible for the program to find the translated content that below there. Would you mind changing it back?

File: web/gds-user-ui/src/locales/en/messages.po [hunk 3]
```diff
@@ -759,7 +786,7 @@ msgstr "Name, Address, Country, National Identifier"
 msgid "National Identification"
 msgstr "National Identification"
 
-#: src/components/OtherJuridictions/index.tsx:40
+#: src/components/OtherJurisdictions/index.tsx:40
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/en/messages.po:789]: See my note above - unfortunately we'll have to live with the filepath typo for now. Would you mind changing it back?

File: web/gds-user-ui/src/locales/en/messages.po [hunk 4]
```diff
@@ -929,7 +956,7 @@ msgstr "Registration Authority cannot be left empty"
 msgid "Registration Submission"
 msgstr "Registration Submission"
 
-#: src/components/OtherJuridictions/index.tsx:48
+#: src/components/OtherJurisdictions/index.tsx:48
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/en/messages.po:959]: Here too

**Review by rebeccabilbro — CHANGES_REQUESTED** [review]
Great start @nwhite222! There are a few things I noticed in the PR that will need updating; see my inline notes and let me know if you have any questions!

File: web/gds-user-ui/src/locales/en/messages.po (hunk 1 above)
Comment (nwhite222) [reply web/gds-user-ui/src/locales/en/messages.po:22]: I've changed that chunk in the .po files - did you also want me to remove _all_ of the lines that start with "#: src", or just this chunk?

File: web/gds-user-ui/src/locales/en/messages.po (hunk 2 above)
Comment (nwhite222) [reply web/gds-user-ui/src/locales/en/messages.po:117]: Understood, I'll change it back.

**Comment by rebeccabilbro** [comment]
@nwhite222 feel free to close this PR and open a fresh one with the translations when you are ready!

**Pull request closed without merging**